routers/cart-router.js

```diff
@@ -0,0 +1,93 @@
+const express = require('express')
+const bodyParser = require('body-parser')
+const {Cart} = require('../models/cart-model')
+const CartRouter   = express.Router()
+CartRouter.use(bodyParser.json({limit: '5000kb'}))
```

Comment:
Similarly for this syed ...`bodyParser.json()` will just put the body into `req`.And we always want to put body into request for every single route .So i  think u should also move it to index.js .